crates/bevy_winit/src/lib.rs

```diff
@@ -578,7 +587,59 @@ fn handle_winit_event(
                             winit_events.send(ReceivedCharacter { window, char });
                         }
                     }
-                    winit_events.send(converters::convert_keyboard_input(event, window));
+
+                    // Send a `KeyboardInput` event if this key is not a modifier.
+                    let keyboard_event = converters::convert_keyboard_input(event, window);
+                    let mod_keys = [Key::Alt, Key::Control, Key::Shift, Key::Super];
+                    if !mod_keys.contains(&keyboard_event.logical_key) {
```

Comment:
Why do we need to make this re-sending conditional? Don't we want to retain the old behavior on all other platforms (which don't send `ModifiersChanged`?

Reply:
Huh sorry I didn't realize `ModifiersChanged` isn't triggered on other platforms. Windows looks to be working again in 
c25df8f

Reply:
Also I left the conditional re-sending on MacOS to avoid duplicating events - `WindowEvent::KeyboardInput` and `ModifiersChanged` seem to be triggered at the same time except with the native overlay scenario